com.ozmartians.VidCutter.json

```diff
@@ -474,17 +474,16 @@
             "pip3 install --prefix=${FLATPAK_DEST} ."
         ],
         "post-install": [
+            "mkdir -p ${FLATPAK_DEST}/share/applications",
             "mkdir -p ${FLATPAK_DEST}/share/fonts",
+            "mkdir -p ${FLATPAK_DEST}/share/icons",
             "cp vidcutter/fonts/* ${FLATPAK_DEST}/share/fonts",
-            "cp -R data/icons/ ${FLATPAK_DEST}/share/icons",
-            "mkdir -p ${FLATPAK_DEST}/share/applications",
-            "cp data/desktop/com.ozmartians.VidCutter.desktop ${FLATPAK_DEST}/share/applications",
-            "mkdir -p ${FLATPAK_DEST}/usr",
-            "ln -sf ${FLATPAK_DEST}/share ${FLATPAK_DEST}/usr"
+            "cp -R data/icons/hicolor ${FLATPAK_DEST}/share/icons",
+            "cp data/desktop/com.ozmartians.VidCutter.desktop ${FLATPAK_DEST}/share/applications"
```

Comment:
```suggestion
            "cp -R data/icons/hicolor ${FLATPAK_DEST}/share/icons/",
            "install -Dm644 -t ${FLATPAK_DEST}/share/applications data/desktop/com.ozmartians.VidCutter.desktop"
```

Reply:
I don't think these are neccessary tbf. Let's see how the current build do. (all previous were cancelled)